internal/bcdb/worldstate_query_processor_test.go

```diff
@@ -1388,19 +1384,19 @@ func TestGetUser(t *testing.T) {
 				},
 			},
 			{
-				name: "querierUser has read-write permission on targetUser",
+				name: "admin always has read permission on targetUser",
```

Comment:
this is the same as the one below it, I think maybe  here you should change the target user:

`Metadata: targetUserMetadataNoACL,` to  `Metadata: targetUserMetadataReadPerm,`

This way you check admin access without ACL and with ACL.Read existing